config/multi_agent_zhong.json

```diff
@@ -1,109 +1,118 @@
 {
-  "scenario": 
-  {
-      "shape": [50, 50],
-      "wrapped": true,
-      "default_delay": "inertial",
-      "default_cell_type": "hoya",
-  
-      "default_state": 
-      {
-          "cell_id": "DefaultCell",
-          "population": 100,
-          "susceptible": 1,
-          "infected": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "recovered": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-          "num_infected": 16,
-          "num_recovered": 30,
-          "initial_infected": 0,
-          "border_length": 1,
-          "land_area": 0,
-          "neighborhood": []
-      },
-  
-      "default_config": 
-      {
-          "hoya": 
-          {
-            "virulence": 0.6,
-            "recovery": 0.4
-          }
-      }
-  },
+    "scenario": {
+        "default_delay": "inertial",
+        "default_cell_type": "zhong",
+        "default_state": {
+            "population_density": 1,
+            "susceptible": 1,
+            "infected": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+            "recovered": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        },
+        "default_vicinity": {
+            "correlation": 0
```

Comment:
I would use as default vicinity 1. It makes no sense having a neighbor which correlation factor is 0, isn't it?